src/app/components/SubjectHeading/SubjectHeading.jsx

```diff
@@ -254,18 +255,19 @@ class SubjectHeading extends React.Component {
     const isMain = (pathname + search).includes(uuid);
     // changes to HTML structure here will need to be replicated in ./SubjectHeadingTableHeader
 
+
     return (
       <React.Fragment>
         <tr
           className={`
             subjectHeadingRow
-            ${(open || children || isMain) ? 'openSubjectHeading' : ''}
-            ${(indentation || 0) === 0 ? 'topLevel' : ''}
-            ${(indentation || 0) !== 0 ? 'nestedSubjectHeading' : ''}
+            ${open && children ? 'openSubjectHeading' : ''}
```

Comment:
It looks like open subject headings will not receive `openSubjectHeading` styling (which could include e.g. bolding) unless they have children. Is that correct and if so, is that intended?

Reply:
The only styling for `openSubjectHeading` is the removal of the bottom border. `.selected` is being used for the bolding. But I did have to change the logic here to use the `narrower` property, because `children` is only when it comes straight from the original API response.